ocs_ci/deployment/baremetal.py

```diff
@@ -577,11 +576,11 @@ def create_pxe_files(self, ocp_version, role):
             extra_data = ""
             bm_install_files_loc = self.helper_node_details["bm_install_files"]
             extra_data_pxe = "rhcos-live-rootfs.x86_64.img coreos.inst.insecure"
-            if ocp_version <= 4.6:
+            if Version.coerce(ocp_version) <= Version.coerce("4.6"):
```

Comment:
Hey @prsurve ,

Can you please use approach from version module I added?
See examples here:
https://github.com/red-hat-storage/ocs-ci/pull/4872/files

See email:
https://mailman-int.corp.redhat.com/archives/ocs-qe/2021-September/msg00292.html